app/src/main/java/com/twilio/video/app/ui/room/ParticipantView.java

```diff
@@ -148,7 +151,15 @@ void setScaleType(int scaleType) {
     }
 
     public void setMuted(boolean muted) {
-        audioToggle.setVisibility(muted ? VISIBLE : GONE);
+        if (audioToggle != null) audioToggle.setVisibility(muted ? VISIBLE : GONE);
+    }
+
+    public void setPinned(boolean pinned) {
+        pinImage.setVisibility(pinned ? VISIBLE : GONE);
```

Comment:
We are saying that `pinImage` is `@Nullable` above. Should we also do the same `null` check that we are doing within `setMuted`?